test/e2e/single_record_test.go

```diff
@@ -79,6 +80,9 @@ var _ = Describe("Single Record Test", Labels{"single_record"}, func() {
 	})
 
 	It("correctly handles wildcard rootHost values", func(ctx SpecContext) {
+		if testDNSProvider == provider.DNSProviderCoreDNS.String() {
+			Skip("coredns does not currently handle wildcard rootHosts correctly!!")
```

Comment:
if we don't have an issue for this we should add one

Reply:
Yes, we need a follow in on issue for this. I know you had changed this, but it was really just pretending to work. The wildcard entry exists, but the other one is lost so its not really working as expected.

Reply:
good spot.

Reply:
https://github.com/Kuadrant/dns-operator/issues/434